x-pack/plugins/reporting/public/management/__test__/constants.ts

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ReportApiJSON } from '../../../common/types';
+import type { ReportMock } from './types';
+
+const buildMockReport = (baseObj: ReportMock) => ({
+  index: '.reporting-2020.04.12',
+  migration_version: '7.15.0',
+  browser_type: 'chromium',
+  max_attempts: 1,
+  timeout: 300000,
+  created_by: 'elastic',
+  kibana_id: '5b2de169-2785-441b-ae8c-186a1936b17d',
+  kibana_name: 'spicy.local',
+  ...baseObj,
+  payload: {
+    browserTimezone: 'America/Phoenix',
+    layout: { dimensions: { height: 720, width: 1080 }, id: 'preserve_layout' },
+    version: '7.14.0',
+    isDeprecated: baseObj.payload.isDeprecated === true,
+    ...baseObj.payload,
+  },
+});
+
+export const mockJobs: ReportApiJSON[] = [
```

Comment:
Since there is nothing specific to `public/management` here, maybe just have this code in `reporting/common`?